README.md

```diff
@@ -23,6 +23,8 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 * **Name**: Name of this DHCP Scope
 * **SubnetMask**: Subnet mask for the scope specified in IP address format
 * **LeaseDuration**: Time interval for which an IP address should be leased
+ * This should be specified in the following format: `Days.Hours:Minutes:Seconds`
```

Comment:
That's a good idea - it's certainly more explicit what the intended timespan is. I'll update the example documentation.